packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1657,3 +1663,31 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
         return this.localCollection.nextInterval(pos);
     }
 }
+
+/**
+ * Information that identifies an interval within a `Sequence`.
+ */
+export interface IntervalLocator {
+    /**
+     * Label for the collection the interval is a part of
+     */
+    label: string;
```

Comment:
i assume we need this because its not possible to get back to the collection from a specific interval? seems like something that could be improved.